### PR TITLE
EPD-2995 |  API Docs v2 |  Create React app, page layout and styles

### PR DESCRIPTION
**Pull request by rob-dumont**
Changes:
- Created new React app w/ CRA, Typescript, Bootstrap
- General folder and component structure
- Added routing
- Added general colors and font styles
- Added rough initial navbar

Notes:
- Created the app within the api docs folder - this will likely be moved as the management of the openAPI files is solidified.

### CODE REVIEW

**Comment by mschembri-TN** [comment]
Task linked: [EPD-2995 API Docs | Create Page Layout and Styles](https://app.clickup.com/t/14197839/EPD-2995)

**Review by simheather — APPROVED** [review]

File: trip-ninja-docs/public/logo192.png
Comment (vickchungg) [inline trip-ninja-docs/public/logo192.png:1]: We probably won't need this and logo512.png

**Review by vickchungg — APPROVED** [review]

File: trip-ninja-docs/public/logo192.png
Comment (rob-dumont) [reply trip-ninja-docs/public/logo192.png:1]: Good call - deleted!

**Comment by simheather** [comment]
localQA Approved!

**Pull request merged**